var/spack/repos/builtin/packages/py-wradlib/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyWradlib(PythonPackage):
+    """wradlib is designed to assist you in the most important steps of
+    processing weather radar data. These may include: reading common data
+    formats, georeferencing, converting reflectivity to rainfall intensity,
+    identifying and correcting typical error sources (such as clutter or
+    attenuation) and visualising the data."""
+
+    homepage = "https://docs.wradlib.org"
+    url      = "https://pypi.io/packages/source/w/wradlib/wradlib-1.5.0.tar.gz"
+
+    version('1.5.0', sha256='9bf0742d7235ea830e83c2269f6b5d1afd83d92696efce0a7bcdb0c4f6604784')
+
+    depends_on('py-setuptools', type='build')
+
+    # https://docs.wradlib.org/en/stable/index.html
+    conflicts('^python@:2.999', when='@1.3:')
```

Comment:
```suggestion
    depends_on('python@:2', when='@:1.2', type=('build', 'run'))
```

Reply:
The documentation only says that they drop support for python2 with version 1.3. I'm not sure if it was not possible to build old version with python3, I'll check this.

Reply:
Since the spack package only provides version 1.5.0, which according to the package documentation requires python3, which is now default, I simply dropped this redundant dependency. Is this okay?

Reply:
Yep, that's okay. If someone needs Python 2 support, they can always add it later.